src/odemis/model/test/remote_test.py

```diff
@@ -39,6 +38,10 @@
 import time
 import unittest
 
+if sys.version_info[0] >= 3:
+    from multiprocessing import Process
+else:
+    from multiprocessing.process import Process
```

Comment:
Actually, you can just write `from multiprocessing import Process`. That also works with Python 2.7.

Reply:
Please just convert that change to a single line. Or is there a reason this cannot be done?

Reply:
sorry, must have overlooked the comment, will pushed update in a minute

Reply:
done